torch_xla/csrc/tensor_util.cpp

```diff
@@ -670,6 +661,15 @@ at::Tensor MakeTensorFromXlaLiteral(const xla::Literal& literal,
   }
 }
 
+bool TensorMemoryCompare(const at::Tensor& t1, const at::Tensor& t2) {
```

Comment:
Undo this change. Leave in anon ns.

Reply:
so we just have tensorCompare to directly call TensorMemoryCompare in all cases?

Reply:
Yes. Leave a comment there telling about non-deterministic ATEM compare in presence of NaN.